src/pages/grants.css

```diff
@@ -1,10 +1,24 @@
 .GrantsCards__Container {
-  display: flex !important;
   flex-wrap: wrap;
   margin-bottom: 40px;
   margin-top: 15px;
-  row-gap: 16px;
-  column-gap: 32px;
+  display: grid !important;
+  grid-template-columns: 48.33% 48.33%;
+  grid-row-gap: 16px;
+  grid-column-gap: 31px;
+}
+
+@media only screen and (min-width: 320px) and (max-width: 990px) {
```

Comment:
Is it possible to swap these properties from the .GrantsCards__Container classes so this media query is only `min-width: 991px` ?